book/http_fundamentals.rst

```diff
@@ -563,7 +575,7 @@ a PHP library that accomplishes two distinct tasks:
 
 The goal of the framework is to integrate many independent tools in order
 to provide a consistent experience for the developer. Even the framework
-itself is a Symfony bundle (i.e. a plugin) that can be configured or replaced
+itself is a Symfony :term:`bundle` (i.e. a plugin) that can be configured or replaced
```

Comment:
Ah no. Looks like https://github.com/symfony/symfony-docs/pull/5464 has not been merged